WikiWatershed/model-my-watershed
# Estimate SedPhos And Recession Coefficient From Rasters

## Overview

- Use the `soilpallland2` raster in the datahub to estimate the mapshed `SedPhos`/`SoilP` value
- Use the `bfi48grd` raster in the datahub to estimate the mapshed `RecessionCoef` value

Connects #1752
Connects #1756  

### Demo
Resulting GMS file

[panther_skin_creek.gms.txt](https://github.com/WikiWatershed/model-my-watershed/files/1311837/panther_skin_creek.gms.txt)

Line 2, pos 2 is not `0.06`
```
1.4.0,0.01,0,10,0,0,0.12504566328764893,13.825475584291853,62287,24254.230851633638,38032.769148366366,1914.431863640184,0,8.460119315139668,0.021151433681949956,28,1963,1990,0.0004415567779477565,7404.395733781011,0,0,0,0.3584968896652178
```

Line 37
```
701.52,816.6336633663367,0.7774450661770334,0.01793083008186061,0.25,0.25
```


## Testing Instructions

- Check out branch
- Restart Celery `vagrant ssh worker -c 'sudo service celeryd restart'`
- Model a WkAoI that isn't cached on the branch and on staging
- Compare line 2, pos 2 and line 37 pos 2 across staging and this branch. They should be different now.
